gsoc-2022-ideas.md

```diff
@@ -269,3 +270,28 @@ Medium: The task does require more than just Bash to Python script conversion. I
 _Duration_: 175 hours or 350 hours
 
 _Mentor_: [@mkurc-ant](https://github.com/mkurc-ant)
+
+
+## Symmetrical placement and routing APIs in OpenFASOC
+
+OpenFASOC is a framework used for automated IC design generation. It sits on top of tools such as OpenROAD primararly but recently is using tools such as gdsfactory and ALIGN. Currently, we have enabled a few py based functions that allow us to help with our analog layout requirements, and we are planning to create more general APIs that could be generalized to new analog designs.
```

Comment:
I suggest using "Python functions" instead of "py functions". It's subtle, but might be confusing for less experienced readers.